solutions/usaco-989.mdx

```diff
@@ -2,46 +2,108 @@
 id: usaco-989
 source: USACO Bronze 2020 January
 title: Race
-author: Sathvik Chundru
+author: Sathvik Chundru, Kevin Sheng
 ---
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_race_bronze_jan20.html)
 
 <LanguageSection>
- 
+<CPPSection>
+
+```cpp
+#include <iostream>
+#include <fstream>
+
+using std::cout;
+using std::endl;
+
+int fastest_time(int dist, int max_speed) {
+	int lhs_travel = 0;  // Amount of distance where Bessie's speeding up
```

Comment:
maybe not follow in the official solution's footsteps and use actual better variable names lmao

Reply:
i added a comment?

Reply:
i still think modifying the actual variable name is more effective because its more prevalent

Reply:
tbh i feel like it makes pretty good sense

Reply:
i think it'll be just better if you swapped the info in the comment and the variable names.
variable names should represent what the thing it stores is used for. lhs and rhs tells me nothing.

Reply:
@ryanchou-dev if you agree i'll switch

Reply:
i agree with @envyaims, the variable names are pretty bad

Reply:
done